Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -25,6 +25,7 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
 
     private var completionActive = false
     private var canAutocomplete = true
+    private var placeCaretAtEnd = false
```

Comment:
I'd rename this to `shouldApplyCompletion` or something. `applyCompletion` does more than just move the caret to the end.